src/main/java/org/sqlite/jdbc3/JDBC3ResultSet.java

```diff
@@ -547,35 +498,7 @@ public Timestamp getTimestamp(int col) throws SQLException {
      * @see java.sql.ResultSet#getTimestamp(int, java.util.Calendar)
      */
     public Timestamp getTimestamp(int col, Calendar cal) throws SQLException {
-        if (cal == null) {
```

Comment:
(Duplicate) It seems like this moves the driver away from the JDBC spec by ignoring the user supplied Calendar/Timezone information.